src/components/modeler/Modeler.vue

```diff
@@ -1918,8 +1918,12 @@ export default {
       const url = '/package-ai/generateProcessArtifacts';
 
       window.ProcessMaker.apiClient.post(url, params)
-        .then(() => {
-          // Response
+        .then((response) => {
+          if (response.data) {
```

Comment:
You can use ioptional chaining here to improve the code reading something like `response.data?.error`